shopify.app.toml

```diff
@@ -5,10 +5,19 @@ client_id = "<API Client ID>"
 application_url = "https://example.com"
 embedded = false
 
+[build]
```

Comment:
My only ask: can we move this to the bottom of the file? It's done that way in our docs :)